src/schemas/v2/combined.json

```diff
@@ -19862,10 +20241,25 @@
                     }
                   },
                   "baspiRequired": ["centralHeatingDetails"],
+                  "ntsRequired": ["centralHeatingDetails"],
                   "ta6Required": ["centralHeatingDetails"]
+                },
+                {
+                  "properties": {
+                    "heatingType": {
+                      "enum": ["Community heating system"]
+                    },
+                    "details": {
+                      "ntsRef": "B3.4.4",
+                      "title": "Community heating system details",
+                      "description": "Please descripbe how the cost of heating supply is charged, whether you have control over the energy supplier and whether you have any control over the heating",
```

Comment:
Small typo in describe:

`"description": "Please describe how the cost of heating supply is charged, whether you have control over the energy supplier and whether you have any control over the heating",`